### PR TITLE
made staircase mode an actual mode for scaffold

### PR DESCRIPTION
**Pull request by grajkogj**
made staircase mode an actual mode for scaffold along with making active modules a hell of a lot easier to tell apart from inactive modules in the click GUI menu

### CODE REVIEW

File: Horion/Module/Modules/Scaffold.h [hunk 1]
```diff
@@ -7,6 +7,8 @@ class Scaffold : public IModule {
 	bool tryScaffold(vec3_t blockBelow);
```
Comment (N0tExisting) [inline Horion/Module/Modules/Scaffold.h:7]: Why is `staircaseMode` still here?

**Comment by MelihGMC2** [comment]
why is this pr still not closed bruh

**Comment by Imrglop** [comment]
> why is this pr still not closed bruh

ikr

**Comment by xDidntPot** [comment]
@N0tExisting

**Comment by Intoprelised** [comment]
> why is this pr still not closed bruh

yeah

**Comment by hrkyr** [comment]
actually close this bruh

**Comment by Drakesdrs** [comment]
close it

**Comment by MrDiamond64** [comment]
finally

**Pull request closed without merging**